### ISSUE TITLE
cite @namebiliography don't work in the label! 

### ISSUE THREAD

**Issue by nqt2022** [ute/custom-numbered-blocks#7]
In your exmaple, 
I insert this (after number-sections: true, and before `filters:`):

bibliography: "link my Pseudo citedrive"

for the bibliography

In your example:

::: TODO
### Custom styles
- [ ] create an API for user defined block styles @geore1997
- [ ] provide an example
- [ ] and documentation
:::

I insert  \ref{TeachersHope} (your code this example) in label:

`### Custom styles \ref{TeachersHope} `

I see: It work ok :)

But, when I cite @name000 (have in my own citedrive) of the bibliography. 

`### Custom styles @name000`

And I see: It don't work only `pdf` but also `html`. :( I only see `@name000` on the pdf and html.

**Comment by ute** [comment]
Hi @nqt2022, Thank you for reporting, and sorry, I did not have time to answer (semester was too busy).
I do not really understand your citedrive bibliography. Does your quarto document work as expected when you do not use the custom-numbered-blocks extension?

**Comment by nqt2022** [comment]
Yes, on `callouts` of `revealjs`, cite very ok for for the bibliography :)
Thanks for your support!

**Comment by ute** [comment]
Yes, it works with callouts :-). 
Quarto folks are working heavily on crossreferenceable ::: blocks, and reprogram the whole cross referencing mechanism.  Numbering environments (and having them collapsible) was the main reason for me to make this extension. Maybe a later version (Quarto 1.5, perhaps) has better tools to achieve this - it will either make custom-numbered-blocks obsolete, or trigger a major rewrite. Then I would look at how to resolve bibliography references before using a header as title of a block. I did that in a very straightforward way.

**Comment by ute** [comment]
I have almost completely rewritten the internal mechanism - now it works like custom callout :-)

**Comment by nqt2022** [comment]
Thanks for your support :) happy a new year 2026

Vào Thứ 5, 8 thg 1, 2026 vào lúc 07:10 Ute Hahn ***@***.***>
đã viết:
